crates/tako/src/worker/rpc.rs

```diff
@@ -403,6 +403,7 @@ async fn worker_message_loop(
                     .insert(msg.worker_id, msg.address)
                     .is_none())
             }
+            //todo: look
```

Comment:
This commit is a bit extra undercooked, but I am putting it out for discussion before I continue.